lib/screens/authentication/login_screen.dart

```diff
@@ -27,8 +28,14 @@ class LoginScreen extends StatelessWidget {
           child: Row(
             mainAxisAlignment: MainAxisAlignment.center,
             children: [
-              Text('Recover your funds ', style: Theme.of(context).textTheme.subtitle2),
-              const Text('here'),
+              Text(
+                'Recover your funds ',
+                style: Theme.of(context).textTheme.subtitle2,
+              ),
+              Text(
+                'here',
```

Comment:
<img width="246" alt="image" src="https://user-images.githubusercontent.com/8041901/193461202-d05d6538-dd92-42ad-8a6c-1e01b2987a48.png">

Reply:
should this even be there? Dont we recover an account after creating an account?

Reply:
Yes this should not be there